easybuild/easyconfigs/g/Geant4/Geant4-11.0.0-GCC-11.2.0-visualization.eb

```diff
@@ -0,0 +1,52 @@
+name = 'Geant4'
+version = '11.0.0'
+versionsuffix = '-visualization'
+
+homepage = 'https://geant4.cern.ch/'
+description = """Geant4 is a toolkit for the simulation of the passage of particles through matter.
+ Its areas of application include high energy, nuclear and accelerator physics,
+ as well as studies in medical and space science."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+github_account = 'Geant4'
+source_urls = [GITHUB_SOURCE]
+sources = [{'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCELOWER_TAR_GZ}]
+checksums = ['98959447637fe7acc52b532e39906e6109845f9df11bb2a7439e6024a4c74e33']
+
+builddependencies = [('CMake', '3.21.1')]
+dependencies = [
+    ('expat', '2.4.1'),
+    # recommended CLHEP version, see https://geant4-data.web.cern.ch/ReleaseNotes/ReleaseNotes.11.0.html
+    ('CLHEP', '2.4.5.1'),
+    # optional:
+    ('libGLU', '9.0.2'),
+    ('Xerces-C++', '3.2.3'),
+    ('Qt5', '5.15.2'),
+    ('Python', '3.9.6'),
+    ('Boost.Python', '1.77.0'),
+    ('Coin', '4.0.0'),
+    ('SoXt', '1.4.0'),
+    ('SoQt', '1.6.0'),
+    ('motif', '2.3.8'),
+    ('freetype', '2.11.0'),
+]
+
+# build options
+enable_freetype = True      # requires freetype
+enable_gdml = True          # requires X11, Xerces-C++
+enable_inventor = False     # requires Coin, SoXt, libGLU
+enable_inventorqt = True    # requires Coin, SoXt, libGLU, Qt5
+enable_opengl = True        # requires libGLU
+enable_python = True        # requires Python3, Boost.Python
+enable_qt5 = True           # requires Qt5, libGLUenable_raytracer = True     # requires X11
+enable_raytracer = True     # requires X11
+enable_toolssg = 'QT'       # choose one of: 'X11', 'XT' (motif), 'QT' (Qt5)
+enable_xm = True            # requires motif
+
+install_data = True
+# install_data_dir = '/custom/path/to/data'
+
+configopts = "-DEXPAT_ROOT_DIR=$EBROOTEXPAT -DCLHEP_ROOT_DIR=$EBROOTCLHEP"
```

Comment:
@jfgrimm Any reason this isn't taken care of by the easyblock, if it sees that `expat` and `CLHEP` are included as dependency?